.readme-partials.yaml

```diff
@@ -227,4 +227,17 @@ custom_content: |
         <artifactId>google-cloud-logging-servlet-initializer</artifactId>
       </dependency>      
       ```
+  #### Population of Log Entrys' Trace/Span ID
```

Comment:
Do we want to title this "Automatic Trace/Span ID Extraction" to be consistent with the language used in other client libraries?

Reply:
Hi Kevin, Java-logging is a bit special here. We don't have auto extraction for Http headers unless it's through manual setting or using servlet-initializer.